lib/internal/url.js

```diff
@@ -150,52 +151,64 @@ function isURLSearchParams(self) {
 }
 
 class URLSearchParams {
+  [searchParams] = [];
+
+  // "associated url object"
+  [context] = null;
+
   // URL Standard says the default value is '', but as undefined and '' have
   // the same result, undefined is used to prevent unnecessary parsing.
   // Default parameter is necessary to keep URLSearchParams.length === 0 in
   // accordance with Web IDL spec.
   constructor(init = undefined) {
-    if (init === null || init === undefined) {
-      this[searchParams] = [];
+    if (init == null) {
```

Comment:
I would like to know the intention of this update since `null == undefined` that seems to be better to be specific.

Reply:
There are 2 reasons why I've changed this line, but if you have a strong opinion about reverting these lines, I'm happy to change.

1. This is pretty much a common pattern in Node.js `lib` folder. According to my IDE: there are a total of 97 ` == null` checks in the Node.js repository. 
2. I **personally** prefer this change, because it's shorter.